client/src/pages/MemberProfilePage/MemberProfilePage.js

```diff
@@ -125,11 +125,10 @@ class MemberProfilePage extends Component {
                   facebook={people[0].facebook}
                   pinterest={people[0].pinterest}
                   type="default"
-                  name={people[0].name}
+                  name={people[0].FirstName + " " + people[0].Surname[0] + "."}
```

Comment:
Maybe this logic should be in the component, since we want it to be applied consistently.